addons/mail/static/src/new/web/chatter.xml

```diff
@@ -50,22 +50,22 @@
                                 </t>
                             </div>
                         </t>
-                        <div t-elif="!thread.hasWriteAccess" class="o-mail-chatter-topbar-follower-list-no-followers dropdown-item disabled">
+                        <div t-elif="!thread.hasWriteAccess" class="dropdown-item disabled">
                             No Followers
                         </div>
                     </t>
                 </Dropdown>
-                <button t-if="thread.followerOfSelf" class="o-mail-chatter-topbar-unfollow btn px-0" t-att-class="{ 'text-success': !unfollowHover.isHover, 'text-warning': unfollowHover.isHover, 'my-2': !props.compactHeight }" t-att-disabled="!props.resId" t-on-click="onClickUnfollow" t-ref="unfollow">
+                <button t-if="thread.followerOfSelf" class="btn px-0" t-att-class="{ 'text-success': !unfollowHover.isHover, 'text-warning': unfollowHover.isHover, 'my-2': !props.compactHeight }" t-att-disabled="!props.resId" t-on-click="onClickUnfollow" t-ref="unfollow">
                     <div class="position-relative">
                         <!-- Hidden element used to set the button maximum size -->
                         <span class="invisible">
                             <i class="fa fa-fw fa fa-eye-slash"/> <t t-out="followingText.length > unfollowText.length ? followingText : unfollowText"/>
                         </span>
-                        <span t-if="unfollowHover.isHover" class="o-mail-chatter-topbar-unfollow-text position-absolute end-0 top-0 bottom-0 text-warning">
-                            <i class="fa fa-fw fa-times"/> <span t-out="unfollowText"/>
+                        <span t-if="unfollowHover.isHover" class="position-absolute end-0 top-0 bottom-0 text-warning">
+                            <i class="fa fa-fw fa-times"/> <span>Unfollow</span>
                         </span>
-                        <span t-if="!unfollowHover.isHover" class="o-mail-chatter-topbar-unfollow-text d-block position-absolute end-0 top-0 bottom-0">
-                            <i class="fa fa-fw fa-check"/> <span t-out="followingText"/>
+                        <span t-if="!unfollowHover.isHover" class="d-block position-absolute end-0 top-0 bottom-0">
+                            <i class="fa fa-fw fa-check"/> <span>Following</span>
```

Comment:
This change "unsync" the text here from the text used above on the "invisible" part, which might lead to issues if we are not careful. (they need to be the same for the hack to work I believe)

Reply:
Ok, I'm reverting this part. I thought it was nice to see the string in template, but this introduce the syncing issue so better revert for now.